src/iceberg/manifest/v1_metadata.cc

```diff
@@ -17,13 +17,12 @@
  * under the License.
  */
 
-#include "iceberg/v1_metadata.h"
-
 #include <memory>
 
 #include "iceberg/json_internal.h"
-#include "iceberg/manifest_entry.h"
-#include "iceberg/manifest_list.h"
+#include "iceberg/manifest/manifest_entry.h"
+#include "iceberg/manifest/manifest_list.h"
+#include "iceberg/manifest/v1_metadata_internal.h"
```

Comment:
Why changing it to internal? I think in most cases internal headers are used to hide symbols from thirdparty libraries.

Reply:
The v1/2/3_metadata.h contains implementations of ManifestEntryAdapter and ManifestFileAdapter, which seems not to be used by the downstream users, so I guess no need to install them.